src/Symfony/Component/Notifier/Message/MessageInterface.php

```diff
@@ -25,4 +25,6 @@ public function getSubject(): string;
     public function getOptions(): ?MessageOptionsInterface;
 
     public function getTransport(): ?string;
+
+    public function transport(string $transport): self;
```

Comment:
Not a big fan, but I don't have any better idea for now.

Reply:
Maybe we could decorate the message with a `DebugMessage` that exposes this information?

Reply:
I like this idea and gave it a try: https://github.com/symfony/symfony/pull/38474 WDYT?